indico/htdocs/sass/base/_pages.scss

```diff
@@ -24,6 +24,7 @@
             display: inline-block;
             margin-bottom: $separator-margin;
             margin-top: 0;
+            max-width: 70%;
```

Comment:
I added a new block to the template to render action buttons inside title, without above style long titles would break the layout.